.github/workflows/react_tests.yml

```diff
@@ -8,33 +8,32 @@ defaults:
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-      with:
-        path: ${{ github.workspace }}/katello
-    - name: Use Node.js
-      uses: actions/setup-node@v1
-      with:
-        node-version: '12.x'
-    # We could update the postinstall action for foreman to look for an environment variable for plugin webpack dirs
-    # before kicking off the ruby script to find them, this would eliminate the ruby dep and running `npm i` in Katello.
-    - uses: ruby/setup-ruby@v1
-      with:
-        ruby-version: 2.6
-    - name: Checkout Foreman repo
-      uses: actions/checkout@v2
-      with:
-        repository: theforeman/foreman
-        path: ${{ github.workspace }}/foreman # checkout Foreman and Katello as siblings, same as dev env
-        ref: develop
-    - name: Install Foreman dependencies
-      run:  npm i
-      working-directory: ${{ github.workspace }}/foreman
-    - name: Install Katello dependencies
-      run:  npm i
-    - name: Run Katello linting
-      run: npm run lint
-    - name: Run Katello tests
-      run: npm test
+      - uses: actions/checkout@v2
+        with:
+          path: ${{ github.workspace }}/katello
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+      # We could update the postinstall action for foreman to look for an environment variable for plugin webpack dirs
+      # before kicking off the ruby script to find them, this would eliminate the ruby dep and running `npm i` in Katello.
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.6
+      - name: Checkout Foreman repo
+        uses: actions/checkout@v2
+        with:
+          repository: Ron-Lavi/foreman
```

Comment:
was this commit intentional?

Reply:
yes, this way i can test it with the core PR (my branch)